jobs/web/spec

```diff
@@ -1449,7 +1449,6 @@ properties:
     example:
       concourse: |
         ssh-rsa key key@pivotal.io
-    default: {}
```

Comment:
Ah interesting, what was the issue with this default? I would expect an empty map to be valid here since it expects a map type. 🤔 Did the templates generate something bogus?

Reply:
yeah, the bosh templating engine turns an empty hash into an empty string, which go-flags does not parse into an empty map!

```
% bosh-template jobs/web/templates/bpm.yml.erb \
  --context '{"properties":{"bind_port":8080, "worker_gateway": {"authorized_keys":{}}}}' | 
  grep AUTH
    CONCOURSE_TSA_AUTHORIZED_KEYS: ""
```